lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.ts

```diff
@@ -117,6 +117,9 @@ export class TaskListCloudComponent extends DataTableSchema implements OnChanges
     @Input()
     multiselect: boolean = false;
 
+    @Input()
```

Comment:
you need to add the doc here on top of the input the docbbuilder get it from here you can also just add it here and run npm run doc-build

Reply:
updated code